semantic/src/site/collections/table.overrides

```diff
@@ -87,3 +87,11 @@
     border-bottom-right-radius: 10px;
   }
 }
+
+// For listBuilder table not overlap the Application/Review
+// Page area - will show scroll when bigger than container
+.ui.table.list-builder {
+  overflow: auto;
+  display: inline-block;
+  max-width: inherit;
+}
```

Comment:
We've been putting plug-in specific CSS in the plugin folder. In fact, the list-builder already has a "styles.less" file in it, so that CSS should go in there.

Reply:
Also, pretty sure those last two properties (display, max-width) are not necessary.